### ISSUE TITLE
Add API for non watermark video to video_info

### ISSUE THREAD

**Issue by Evil0ctal** [krypton-byte/tiktok-downloader#12]
Is that passbile to add this website as one of the API?
Link: https://ttvideodownload.com/
It has no water mark video link

**Comment by krypton-byte** [comment]
Nope, there is a signature or token or something like that. If you can scrap that website, you can pull requests at this repo.

**Comment by Evil0ctal** [comment]
> 

Yeah, I saw that too, Im trying to use selenium to scrape it and deploy on heroku. I will make an API when i done.

**Comment by Evil0ctal** [comment]
The website has a limit for the request per day by per IP, so won't be able to use it as API, or need to use proxy to scrape it.

Here is a short code for scrape:

import time
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By
from webdriver_manager.chrome import ChromeDriverManager


def get_video_info(link):
    api_url = "https://ttvideodownload.com/"
    driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))
    driver.get(api_url)
    # Find input box
    search_box = driver.find_element(By.CLASS_NAME, "form-control")
    search_box.send_keys(tiktok_link)
    time.sleep(3)
    # Find submit button
    search_button = driver.find_element(By.CLASS_NAME, "mb-hidden")
    search_button.click()
    print(driver.get_log('browser'))
    time.sleep(10)


if __name__ == '__main__':
    tiktok_link = "https://www.tiktok.com/@hoodvineunrated/video/7039805708220501294"
    get_video_info(tiktok_link)

**Comment by thucngv** [comment]
I have demo tiktok original api to get video url without watermark, please refer to the following repository
https://github.com/thucngv/tiktok-downloader

**Comment by ZefianAlfian** [comment]
> I have demo tiktok original api to get video url without watermark, please refer to the following repository https://github.com/thucngv/tiktok-downloader

404 not found

**Comment by Evil0ctal** [comment]
> > I have demo tiktok original api to get video url without watermark, please refer to the following repository https://github.com/thucngv/tiktok-downloader
> 
> 404 not found

Try my repo:
https://github.com/Evil0ctal/Douyin_TikTok_Download_API
